### PR TITLE
Bulkloading

### PR DESCRIPTION
**Pull request by Deepak-Vohra**
Added page on bulk loading.

### CODE REVIEW

File: doc/source/operating/bulk_loading.rst [hunk 1]
```diff
@@ -1,24 +1,693 @@
...
+Bulk Loading from a Snapshot
++++++++++++++++++++++++++++++ 
+In this section we shall demonstrate restoring a snapshot of the ``magazine`` table to the ``magazine`` table.  As we used the same table to restore data from a backup the directory structure required by ``sstableloader`` should already exist.  If the directory structure needed to load SSTables to ``catalogkeyspace.magazine`` does not exist create the directories and set their permissions.
+
+::
+
+ [ec2-user@ip-10-0-2-238 ~]$ sudo mkdir -p /catalogkeyspace/magazine
+ [ec2-user@ip-10-0-2-238 ~]$ sudo chmod -R 777 /catalogkeyspace/magazine
+
+As we shall be copying the snapshot  files to the directory remove any files that may be in the directory.
```
Comment (alexott) [inline doc/source/operating/bulk_loading.rst:223]: it's easier just to create symlinks, not copy the data. something like:

```sh
mkdir keyspace_name
ln -s _path_to_snapshot_folder keyspace_name/table_name
```

File: doc/source/operating/bulk_loading.rst [hunk 2]
```diff
@@ -1,24 +1,693 @@
...
+
+Using nodetool import
+^^^^^^^^^^^^^^^^^^^^^
+In this section we shall import SSTables into a table using the ``nodetool import`` command. The ``nodetool refresh`` command is deprecated, and it is recommended to use ``nodetool import`` instead. The ``nodetool refresh`` does not have an option to load new SSTables from a separate directory which the ``nodetool import`` does.
+
+The command usage is as follows.
+
+::
+
+ nodetool <options> -- <keyspace> <table> <directory> ...
```
Comment (alexott) [inline doc/source/operating/bulk_loading.rst]: `nodetool <options> import` or `nodetool import <options>` ?

**Comment by Deepak-Vohra** [comment]
Alex, 
Thanks for suggestions.
Updated PR to add using symlinks as an alternative, and fix the _nodetool import_ syntax.

**Review by alexott — APPROVED** [review]
LGTM

**Pull request closed without merging**